src/components/manage/Sharing/__snapshots__/Sharing.test.jsx.snap

```diff
@@ -80,9 +80,7 @@ exports[`Sharing renders a sharing component 1`] = `
             </th>
             <th
               className=""
-            >
```

Comment:
@cekk this seems to be unrelated to the contact-info view (same for the translation). Did you do this by purpose or accident?

Reply:
Yes, it's unrelated, but i had this broken test and without this change i can't had green lights